OpenRA.Mods.Common/Graphics/DefaultSpriteSequence.cs

```diff
@@ -39,20 +38,8 @@ public virtual ISpriteSequence CreateSequence(ModData modData, TileSet tileSet,
 			}
 
 			foreach (var kvp in nodes)
-			{
 				using (new Support.PerfTimer("new Sequence(\"{0}\")".F(node.Key), 20))
-				{
-					try
-					{
-						sequences.Add(kvp.Key, CreateSequence(modData, tileSet, cache, node.Key, kvp.Key, kvp.Value));
-					}
-					catch (FileNotFoundException ex)
-					{
-						// Eat the FileNotFound exceptions from missing sprites
-						OnMissingSpriteError(ex.Message);
```

Comment:
You can do this by keeping the try/catch and utility command, and replacing the default OnMissingSpriteError with a throw statement.